components/Member/RegisterMemberItem.js

```diff
@@ -131,7 +131,6 @@ class RegisterMemberItem extends Component {
           {options.btns.map((btn, index) => {
             return (
               <span key={index}>
-                {btn.msg && <p>{btn.msg}</p>}
```

Comment:
I don't think the button message should be removed - it's not part of the story and has important functionality. 
If it should be removed you have not removed the btn.msg data elsewhere in the file. So are suppressing output but leaving all the remaining code in. 

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1596437/70653900-32ccd180-1cba-11ea-86f0-753760c2edee.png">

Reply:
The button message was removed in the new designs that @waltzaround  provided, please discuss with @waltzaround if you wish to leave this in the new design and let me know and I can change this back.